solr/core/src/test/org/apache/solr/schema/ChangedSchemaMergeTest.java

```diff
@@ -83,15 +84,15 @@ private void addDoc(SolrCore core, String... fieldValues) throws IOException {
   }
 
   private CoreContainer init() throws Exception {
-    File changed = new File(solrHomeDirectory, "changed");
-    copyMinConf(changed, "name=changed");
+    Path changed = solrHomeDirectory.toPath().resolve("changed");
+    copyMinConf(changed.toFile(), "name=changed");
     // Overlay with my local schema
-    schemaFile = new File(new File(changed, "conf"), "schema.xml");
-    FileUtils.writeStringToFile(schemaFile, withWhich, StandardCharsets.UTF_8);
+    schemaFile = changed.resolve("conf").resolve("schema.xml").toFile();
+    Files.writeString(schemaFile.toPath(), withWhich, StandardCharsets.UTF_8);
```

Comment:
The fields `solrHomeDirectory` and `schemaFile` should both be changed to be of type Path.  I took a look and we're always using them where we actually want a Path.

Reply:
gah good call. I figured schemaFile and solrHomeDirectory were part of the test framework and not private variables.

Reply:
Fixed in 2d79b286b652e6e5da265a9e3767252fa7063532